cartography/intel/aws/redshift.py

```diff
@@ -0,0 +1,118 @@
+import logging
+
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+def get_redshift_cluster_data(boto3_session, region):
+    client = boto3_session.client('redshift', region_name=region)
+    paginator = client.get_paginator('describe_clusters')
+    clusters = []
+    for page in paginator.paginate():
+        clusters.extend(page['Clusters'])
+    return clusters
+
+
+def _make_redshift_cluster_arn(region, aws_account_id, cluster_identifier):
+    return f'arn:aws:redshift:{region}:{aws_account_id}:cluster:{cluster_identifier}'
+
+
+@timeit
+def transform_redshift_cluster_data(clusters, region, current_aws_account_id):
```

Comment:
will you ever call `get_redshift_cluster_data` without this?  Aka: is the arn mandatory? If it is, it might be worth moving this into `get_redshift_cluster_data` so the functionality is more tightly coupled.  Otherwise people might not realize that they need to do this.

This method also really doesn't need a timeit as you're just doing string formatting.  There is no real way to make this faster and it also shouldn't take up much time.  If you really want to optimize, you can explore parallelizing calls using gevent, asyncio or whatever framework pleases you in `load_redshift_cluster_data`

Reply:
Yeah good call, I'll remove. I got too eager with copypasta.